packages/spanner/src/codec.js

```diff
@@ -26,6 +26,10 @@ var commonGrpc = require('@google-cloud/common-grpc');
 var is = require('is');
 
 function SpannerDate(value) {
+  if (is.number(value)) {
```

Comment:
I don't think this is what we want... I think this should choke on more than one argument. For example:

```js
> new Date(1490195461000)
2017-03-22T15:11:01.000Z
```

I'm worried about people passing the arguments to a "date":

```js
> new Date(2000, 99, 41)
2008-05-11T04:00:00.000Z 
```

So this is really about anything more than one argument.